changelogs/current.yaml

```diff
@@ -21,6 +21,9 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: grpc_json_transcoder
+  change: |
+    Support grpc reflection for building json transcoding.
```

Comment:
Is this a new feature or a modified behavior?

Reply:
It depends on the Envoy project's definition of a feature, so I'm not sure.
If I had to guess, I would guess it's a new feature because it enables a new capability that didn't exist before.

Reply:
In this case this update should probably be under a different category.

Reply:
It is a new feature

Reply:
Thanks for confirming! Will move.